frontend/src/modules/shared/actions.js

```diff
@@ -45,6 +47,8 @@ export const fetchInitialData = () => async (dispatch, getState) => {
         Libraries.listTable,
         Projects.listTable,
         ProcessMeasurements.listTable,
+        Datasets.listTable,
+        DatasetFiles.listTable,
```

Comment:
Not sure this is useful. I don't think we'll ever list all files without first going through the Datasets.

Reply:
If I don't fetch them initially, then it will be difficult to load the datasets into the table. PaginatedTable always assumes that the models have been fetched already.